waku/v2/protocol/store/options.go

```diff
@@ -115,6 +116,13 @@ func IncludeData(v bool) RequestOption {
 	}
 }
 
+func SkipRateLimit() RequestOption {
```

Comment:
might be a good idea to have an option to set rate-limit as well?
not sure how useful it will be, but i am wondering if in the fleet rate-limit is increased does it makes sense to have this as an option as well.